src/main/java/com/jcabi/dynamodb/maven/plugin/CreateTablesMojo.java

```diff
@@ -66,6 +67,7 @@
  * @version $Id$
  * @checkstyle ClassDataAbstractionCoupling (300 lines)
  * @checkstyle MultipleStringLiterals (300 lines)
+ * @checkstyle ExcessiveImports (400 lines)
```

Comment:
@piddubnyi just use search option in GitHub: https://github.com/search?utf8=%E2%9C%93&q=ExcessiveImports+user%3Ateamed&type=Code&ref=searchresults